_posts/2019-11-27-cockpit-208.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Cockpit 208
+author: pitti
+date: 2019-11-27
+tags: cockpit linux
+slug: cockpit-208
+category: release
+summary: Cockpit with Machines, file system creation, and crash reporting improvements
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly.  Here are the release notes from version 208.
+
+### Storage: Drop "default mount point" concept
+
+Cockpit no longer offers the confusing "Default" choice when mounting a filesystem. Now every new filesystem requires the specification of a mount point. Additionally, Cockpit hides the distinction between configuration (`/etc/fstab`) and run-time state (`/proc/mounts`). Changes made in Cockpit always apply to both the configuration and run-time state. When configuration and run-time state differ from each other, Cockpit will warn about that and will let you bring them back in sync.
+
+![Storage format dialog](/images/storage-format-mountpoint.png)
+
+### Machines: Support transient virtual networks and storage pools
+
+Transient resources are resources which exist only while they are running and cease to exists once they stop running/are deactivated. Cockpit now disables or hides functionality like autostart or deletion for transient resources, as these would fail:
```

Comment:
This reads like the resource itself is running.  Is it rather the associated machine is running?

Reply:
I updated the description.